packages/fast-components-styles-msft/src/paragraph/index.ts

```diff
@@ -3,16 +3,22 @@ import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
 import { ParagraphClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
 import { fontWeight } from "../utilities/fonts";
 
+function applyParagraphStyles(): CSSRules<DesignSystem> {
+    return {
+        fontWeight: `${fontWeight.normal}`,
+    };
+}
+
 const styles: ComponentStyles<ParagraphClassNameContract, DesignSystem> = {
     paragraph: {
         "&$paragraph__1": {
-            fontWeight: `${fontWeight.semilight}`,
+            ...applyParagraphStyles(),
```

Comment:
if this is just `font-weight: normal` you can probably just apply this to `paragraph` and not apply it to each level *of* paragraph.